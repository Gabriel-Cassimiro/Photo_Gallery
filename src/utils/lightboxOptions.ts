export const Options = {
	settings: {},
	caption: {},
	buttons: {},
	thumbnails: {},
	progressBar: {}
}
