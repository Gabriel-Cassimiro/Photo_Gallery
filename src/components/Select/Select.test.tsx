/**
 * @jest-environment jsdom
 */
import React from "react"
import { render } from "@testing-library/react"
import { Select } from "./index"
import { TestProvider } from "../../tests/utils"

describe("Test Select component", () => {
	function Element() {
		return (
			<TestProvider>
				<Select />
			</TestProvider>
		)
	}

	it("Should render correctly", () => {
		const { container } = render(<Element />)
		expect(container).toMatchSnapshot()
	})
})
