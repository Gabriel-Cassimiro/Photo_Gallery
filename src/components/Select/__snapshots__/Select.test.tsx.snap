// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Select component Should render correctly 1`] = `
<div>
  <div
    class="w-auto ml-auto"
  >
    <label
      class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
      for="select-pages"
    >
      Photos per page
    </label>
    <select
      class=" w-full font-bold border border-gray-200 text-gray-700 py-2 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
      id="select-pages"
    >
      <option
        selected=""
        value="20"
      >
        20
      </option>
      <option
        value="40"
      >
        40
      </option>
      <option
        value="60"
      >
        60
      </option>
      <option
        value="80"
      >
        80
      </option>
    </select>
  </div>
</div>
`;
